.appveyor.yml

```diff
@@ -58,8 +58,8 @@ for:
         if "%LLVM_VERSION%"=="latest" (set WASM=ON)
         if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2019" ( (set generator="Visual Studio 16") & (set vspath="C:\Program Files (x86)\Microsoft Visual Studio\2019"))
         if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2022" ( (set generator="Visual Studio 17") & (set vspath="C:\Program Files\Microsoft Visual Studio\2022"))
-        set LLVM_TAR=llvm-14.0.3-win.vs2022-Release+Asserts-x86.arm.wasm.tar.7z
-        if "%LLVM_VERSION%"=="13.0" (set LLVM_TAR=llvm-13.0.1-win.vs2022-Release+Asserts-x86.arm.wasm.tar.7z)
+        set LLVM_TAR=llvm-14.0.3-win.vs2019-Release+Asserts-x86.arm.wasm.tar.7z
```

Comment:
Should the link to the latest Windows binaries be updated in Readme.md after this change?

Reply:
Good catch!!! I was using VS2022 image to build ISPC, while I was using LLVM built with VS2019. And it did work miraculously... While it was not working when using VS2022 across the board... Grrrrr.
I pushed commit to use VS2019 image and fixed the download link.